tests/Autodesk.Forge.Core.Test/TestForgeAgentHandler.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Moq;
+using Moq.Protected;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using Xunit;
+
+namespace Autodesk.Forge.Core.Test
+{
+    public class TestForgeAgentHandler
+    {
+        [Fact]
+        public async void TestUser()
+        {
+            var json = @"
+            {
+                ""Forge"" : {
+                    ""ClientId"" : ""bla"",
+                    ""ClientSecret"" : ""blabla"",
+                    ""Agents"" : {
+                        ""user1"" : {
+                            ""ClientId"" : ""user1-bla"",
+                            ""ClientSecret"" : ""user1-blabla""
```

Comment:
so we don't have to define `Agents` for an array of users, but if `Agents` are defined, the user in it will be used instead
Can we update this example usage in https://github.com/Autodesk-Forge/forge-api-dotnet-core/blob/main/README.md?

Reply:
`Agents` will only be used if the `ForgeHandler` receives an "Agent" in `HttpRequestMessage.Options`.
We don't currently document any example usage in the readme because we don't expect people to use `Autodesk.Forge.Core` directly. We expect that they use `Autodesk.Forge.DesignAutomation` instead.

Reply:
Sure, we should update readme.md in Autodesk.Forge.DesignAutomation then